scipy/special/tests/test_mpmath.py

```diff
@@ -360,6 +360,32 @@ def test_loggamma_taylor2():
 
 
 # ------------------------------------------------------------------------------
+# rgamma
+# ------------------------------------------------------------------------------
+
+def test_rgamma_zeros():
```

Comment:
Also, all the test routines using mpmath (apart from in TestSystematic) should be decorated with `@mpmath_check('0.19')`  (using e.g. whatever version number you are sure the functionality is in).